packages/core/src/anchor/__tests__/anchor-processing-loop.test.ts

```diff
@@ -152,7 +152,7 @@ test('Dont process the same entry multiple times concurrently', async () => {
   async function* overEntries(): AsyncGenerator<number> {
     do {
       // Need this sleep or else the Node runtime might never switch back to the "process" function.
-      await CommonTestUtils.delay(1)
+      await CommonTestUtils.delay(5)
```

Comment:
@ukstv This is how I solved it

Reply:
what's the issue you were seeing?  This change really shouldn't be necessary.  This sleep isn't actually making the test wait for anything to happen, it's just so that the Node event loop has an interrupt point so other jobs can get scheduled, otherwise the generator can just emit events forever without the task to process those events ever running.  But increasing the duration of the sleep should have no affect on the rest of the system or the test, unless I really misunderstood something when I wrote this test.

Reply:
I'm assuming this test failure was the issue: https://app.circleci.com/pipelines/github/ceramicnetwork/js-ceramic/13094/workflows/f7e8914c-f718-40e3-bfd6-4ee07174026c/jobs/18766

Put together a PR that I believe should fix the race condition in the test: https://github.com/ceramicnetwork/js-ceramic/pull/3192

Reply:
Can this change be reverted now that https://github.com/ceramicnetwork/js-ceramic/pull/3192 is in?

Reply:
ping